bash.fsr/AddQuoteVC.swift

```diff
@@ -41,6 +43,7 @@ class AddQuoteVC: UIViewController {
 				self.presentViewController(alert, animated: true, completion: nil)
 			}
 		}
+        self.navigationItem.rightBarButtonItem?.action = Selector("addQuote")
```

Comment:
You can also do this in Swift via defer:
```swift
defer { self.navigationItem.rightBarButtonItem?.action = Selector("addQuote") }
```
Also, the syntax for selectors is changing with the upcoming version of Swift, just FYI: 
```swift
#selector(self.addQuote())
``` 
no more strings \o/